test-cases/longevity/longevity-lwt-basic-24h.yaml

```diff
@@ -1,19 +1,59 @@
-test_duration: 1500
-prepare_write_cmd: ["cassandra-stress user profile=/tmp/c-s_lwt_basic.yaml n=10000000 ops'(insert_query=1)' cl=QUORUM -mode native cql3 -rate threads=30" ]
-stress_cmd: ["cassandra-stress user profile=/tmp/c-s_lwt_basic.yaml ops'(lwt_update_one_column=1,lwt_update_two_columns=1)' cl=QUORUM duration=1440m -mode native cql3 -rate threads=20",
-             "cassandra-stress user profile=/tmp/c-s_lwt_basic.yaml ops'(lwt_update_one_column=1,lwt_update_two_columns=1)' cl=QUORUM duration=1440m -mode native cql3 -rate threads=20"
-            ]
-stress_read_cmd: ["cassandra-stress user profile=/tmp/c-s_lwt_basic.yaml ops'(select=1)' cl=SERIAL duration=1440m -mode native cql3 -rate threads=20" ]
+test_duration: 360
+
+prepare_write_cmd:
+  - >-
+    latte run
+    -f lwt_insert
+    -P rows_per_partition=1000 \
+    -P partition_sizes="\"100:1\""
+    --rate 50000
+    --duration=10000000
+    data_dir/latte/lwt_load.rn
+
+
+
+stress_cmd:
+  # TODO: what loader instance should be used? Its not clear needs to be clarified
+  - >-
+    latte run -f lwt_upd_one --threads=7 --concurrency=128 --duration=80m
+    data_dir/latte/lwt_load.rn
+
+  - >-
+    latte run -f lwt_upd_two --threads=7 --concurrency=128 --duration=80m
+    data_dir/latte/lwt_load.rn
+
+  - >-
+    latte run -f lwt_read --consistency=QUORUM --threads=7 --concurrency=128 --duration=80m
+    data_dir/latte/lwt_load.rn
+
+  - >-
+    latte run -f lwt_delete_partitions -P delete_pct=50 --duration=80m
```

Comment:
I don't know the requirements for your case,
but I feel that such a command, that deletes data, better be explicitly rate limited to be predictable for 100%.

The same about read commands, having rate limit for each you will know for sure when we read and delete and which partitions.

JFYI: Using the partition-row distribution preset feature from latte we distribute stuff evenly as balanced as possible.
It means that in current first 10000 calls we get 10000 partitions with 1 row and only starting with 10001 we go over the partitions again having row diff be only 1.